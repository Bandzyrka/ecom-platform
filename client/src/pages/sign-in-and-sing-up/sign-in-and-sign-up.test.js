import React from 'react';
import { shallow } from 'enzyme';
import SignInAndSignUp from './sign-in-and-sign-up.component';

it('should render SignInAndSignUp page', () => {
  expect(shallow(<SignInAndSignUp />)).toMatchSnapshot();
});   