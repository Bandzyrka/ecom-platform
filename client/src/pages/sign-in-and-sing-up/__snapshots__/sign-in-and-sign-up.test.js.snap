// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SignInAndSignUp page 1`] = `ShallowWrapper {}`;
